### ISSUE TITLE
Implement `ockam service list` command 

### ISSUE THREAD

**Issue by mrinalwadhwa** [build-trust/ockam#4094]
## Currently

There is no command in to list a node's services. There is, though, [an endpoint](https://github.com/build-trust/ockam/blob/7cbe20ebbeeb7515ae63aae6804581f0fbb20044/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L553) on the `NodeManager` to retrieve the current services stored in its internal `registry`.

## Desired

Every `NodeManager` has a registry where all the initialized services are stored. A new command should be added on `ockam_command::service` to list the services running on a particular node.
  
To do that, the following must be done:

- On the `ockam_command` crate, implement the `Output` trait for the [`ServiceList`](https://github.com/build-trust/ockam/blob/7cbe20ebbeeb7515ae63aae6804581f0fbb20044/implementations/rust/ockam/ockam_api/src/nodes/models/services.rs#L376).
- Also on the `ockam_command` crate, add a new subcommand called `service list` [here](https://github.com/build-trust/ockam/blob/7cbe20ebbeeb7515ae63aae6804581f0fbb20044/implementations/rust/ockam/ockam_command/src/service/mod.rs#L18) that will send a request to the [`node/services` endpoint](https://github.com/build-trust/ockam/blob/7cbe20ebbeeb7515ae63aae6804581f0fbb20044/implementations/rust/ockam/ockam_api/src/nodes/service.rs#L553) and print the request's result.

The command logic should be pretty simple, something like the following:

```rust
let mut rpc = RpcBuilder::new(ctx, &opts, &node_name).build();
rpc.request(...).await?;
rpc.parse_and_print_response::<ServiceList>()?;
```

---

_We love helping new contributors!_
_If you have questions or need help as you work on your first Ockam contribution, please leave a comment on [this discussion](https://github.com/ockam-network/ockam/discussions/1081)._
_If you're looking for other issues to contribute to, checkout [this discussion](https://github.com/build-trust/ockam/discussions/3206) and labels - https://github.com/build-trust/ockam/labels/good%20first%20issue or https://github.com/build-trust/ockam/labels/help%20wanted_

**Comment by nidnogg** [comment]
I'd like to have a look into it

**Comment by nidnogg** [comment]
Added a [PR](https://github.com/build-trust/ockam/pull/4141) with a possible solution. Couple of comments:
- I figured `service list` would work by providing the `node_name` as a parameter. Is that all right or should there be a default case where the services for the `default` node are listed instead as well?
- For displaying, I tinkered with a `Table` option for readability, but I opted for rendering the list in the same way as it's done in the `node list` command in the end. Here's a side-by-side comparison of both if it's of any use:
![2023-01-26_23-09](https://user-images.githubusercontent.com/17261185/214995562-71b862bb-51c7-4a0d-9314-901d659bfab0.png)
